frontend/src/services/http/http.service.ts

```diff
@@ -83,11 +83,19 @@ class Http {
       });
     }
 
+    if (response.status === 204) {
+      return null;
+    }
+
     return response;
   }
 
-  private parseJSON<T>(response: Response): Promise<T> {
-    return response.json();
+  private async parseJSON<T>(response: Response | null): Promise<T> {
+    if (response) {
+      return response.json();
+    }
+
+    return null as unknown as T;
```

Comment:
```suggestion
  private async parseJSON<T>(response: Response | null): Promise<T | null> {
    if (response) {
      return response.json();
    }

    return null;
```

Reply:
If update return type to 'T|null' there will be too many changes to be made. Would need to change return type for 'load' method to T|null and that method is used in most of services meaning would need to update them to and so on...

Reply:
Please remove a checker null for 204 status here and add a boolean for delete route response